### PR TITLE
Minor updates to the deploy documentation

### PR DESCRIPTION
**Pull request by tillrohrmann**
Minor updates to the deploy documentation.

### CODE REVIEW

File: docs/deploy/restate_cloud.md [hunk 1]
```diff
@@ -7,62 +7,65 @@ import Admonition from '@theme/Admonition';
 
 # Restate Cloud
```
Comment (tillrohrmann) [inline docs/deploy/restate_cloud.md:8]: @jackkleeman is the cloud documentation page up to date with the actual cloud state? The changes in this PR are only of cosmetic nature.

Comment (jackkleeman) [reply docs/deploy/restate_cloud.md:8]: i dont really think we should have a cloud docs page right now, this is about the old cloud POC

**Review by gvdongen — APPROVED** [review]
LGTM

File: docs/deploy/restate_cloud.md (hunk 1 above)
Comment (gvdongen) [reply docs/deploy/restate_cloud.md:8]: @jackkleeman should we remove it?

Comment (jackkleeman) [reply docs/deploy/restate_cloud.md:8]: id say yes just remove the page and we will write something new for *actual* cloud towards launch

Comment (tillrohrmann) [reply docs/deploy/restate_cloud.md:8]: Ok, will remove it then and create a follow-up ticket to add the new documentation.

File: docs/deploy/overview.md [hunk 2]
```diff
@@ -7,45 +7,47 @@ import Admonition from '@theme/Admonition';
...
+| Ingress   | 8080 | Acts as an API gateway for all services registered with Restate                                                                |                                               |
+| Admin     | 9070 | Allows for CRUD operations on service/service deployment metadata, eg for service registration                                 | REST                                          |
+| Postgres  | 9071 | Exposes Restate RocksDB read-only storage operations using the Postgres protocol. See [Introspection](/operate/introspection). | Postgres                                      |
 
 ## Restate services
 
-Restate services are deployed within _Service deployments_. The Restate runtime interacts with service deployments by sending requests to them using a custom protocol on top of HTTP.
+Restate services are deployed as _Service deployments_.
+The Restate runtime interacts with service deployments via HTTP using Restate's [service protocol](https://github.com/restatedev/service-protocol).
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/deploy/overview.md:44]: 👍

File: docs/deploy/restate_cloud.md [hunk 3]
```diff
@@ -7,62 +7,65 @@ import Admonition from '@theme/Admonition';
...
 
-If you're using Restate with a function as a service platform, it can be challenging to 
-continuously run the runtime in your infrastructure. To make Restate easier to test out,
-we offer a managed cluster to users by request. 
+If you're using Restate with a function as a service platform, it can be challenging to continuously run the runtime in your infrastructure.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/deploy/restate_cloud.md:10]: Will remove the cloud page.

File: docs/deploy/restate_cloud.md [hunk 4]
```diff
@@ -7,62 +7,65 @@ import Admonition from '@theme/Admonition';
...
-
-## SLA
-Restate Cloud is intended to help people try out Restate, and **should never be used in
-production. We offer no promises of data durability or uptime.**
-
 ## Using your cluster
-A managed service comprises of a url like `yourcluster.dev.restate.cloud` and a JWT token. This
-token will be needed for all interactions with the cluster.
+
+A managed service comprises of a url like `yourcluster.dev.restate.cloud` and a JWT token.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/deploy/restate_cloud.md:19]: Will remove the cloud page.

File: docs/deploy/restate_cloud.md (hunk 1 above)
Comment (tillrohrmann) [reply docs/deploy/restate_cloud.md]: for posterity: https://github.com/restatedev/documentation/issues/357

**Pull request merged**